src/aura/STG_CMP_Affl/STG_CMP_Affl.cmp

```diff
@@ -20,82 +29,260 @@
         
         <div aura:id="settsTabContent" class="slds-tabs__content" role="tabpanel">
             <div class="slds-grid slds-wrap">
+			    <div class="slds-col slds-grid slds-m-top--medium">
```

Comment:
I'd like to verify that the indentation here is not an issue with github. I realize it's only a style item but this indentation looks a bit too deep.